joint_teapot/teapot.py

```diff
@@ -89,6 +89,8 @@ def add_all_canvas_students_to_teams(self, team_names: List[str]) -> None:
     def create_personal_repos_for_all_canvas_students(
         self, suffix: str = ""
     ) -> List[str]:
+        if suffix != "" and not suffix.startswith("-"):
+            suffix = "-" + suffix
```

Comment:
I'm getting name(s) of submitter(s) from repo name. It would be much easier to do so if there is a hyphen between name + id/group name (get from canvas) and the suffix. For example, if repo name is "team2p2" I need to write extra codes to find out where to split, and it is not easy to adapt if any TA use strange name/suffix in the future.

Anyway, making sure that a hyphen exists does no bad and makes it cleaner